tests/Unit/CrudPanel/BaseDBCrudPanelTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Backpack\CRUD\Tests\Unit\CrudPanel;
+
+use Backpack\CRUD\CrudPanel;
+use Orchestra\Database\ConsoleServiceProvider;
+
+abstract class BaseDBCrudPanelTest extends BaseCrudPanelTest
+{
+    /**
+     * @var CrudPanel
+     */
+    protected $crudPanel;
+
+    protected $model;
+
+    /**
+     * Setup the test environment.
+     *
+     * @return void
+     */
+    protected function setUp()
+    {
+        parent::setUp();
+
+        // call migrations specific to our tests
+        $this->loadMigrationsFrom([
+            '--database' => 'testing',
```

Comment:
That seems to be a reasonable name and I doubt anyone would have a production database called 'testing' (but I've seen stupider things TBH).

Reply:
The database is an in memory one, first created when you run the tests and destroyed immediately after.
I've configured this [here](https://github.com/Laravel-Backpack/CRUD/pull/930/files#diff-1f8df59f532c3641c2e5e010b4d8fa1fR43) using [orchestral's testbench](https://github.com/orchestral/testbench) database helper. So there should be no problem.

Also, I was thinking of suggesting we add an git ignore entry for the tests folder on the release branch. Thoughts?

Reply:
Ah, missed that.